examples/website-demos/social-messaging/src/components/ThreadReply/ThreadReply.tsx

```diff
@@ -17,29 +13,32 @@ export const ThreadReply: React.FC = () => {
   return (
     <>
       {message.thread_participants ? (
-      <div className='message-wrapper-inner-reply'>
-        {!myMessage && (
-          <>
-            <RightReply />
-            {uniqueThreadParticipants[0] && uniqueThreadParticipants.map((user, i) => (
-              <div className={`${uniqueThreadParticipants[1] ? `left-avatar-${i+1}` : ''}`}>
-                <Avatar key={user.id} size={16} image={user.image} />
-              </div>
-            ))}
-          </>
-        )}
-        <MessageRepliesCountButton onClick={handleOpenThread} reply_count={message.reply_count} />
-        {myMessage && (
-          <>
-            {uniqueThreadParticipants[0] && uniqueThreadParticipants.reverse().map((user, i) => (
-              <div className={`${uniqueThreadParticipants[1] ? `right-avatar-${i+1}` : ''}`}>
-                <Avatar key={user.id} size={16} image={user.image} />
-              </div>
-            ))}
-            <LeftReply />
-          </>
-        )}
-      </div> ) : null}
+        <div className='message-wrapper-inner-reply'>
+          {!myMessage && (
+            <>
+              <RightReply />
+              {uniqueThreadParticipants[0] &&
+                uniqueThreadParticipants.map((user, i) => (
+                  <div className={`${uniqueThreadParticipants[1] ? `left-avatar-${i + 1}` : ''}`}>
+                    <Avatar key={user.id} size={16} image={user.image} />
```

Comment:
add the name field to both Avatar components so initials show in cases of rendering the fallback component, add like...

`name={user.name || user.id}`

Reply:
looks like you added to the bottom but not the top Avatar component